packages/fxa-settings/src/components/Settings/Security/index.tsx

```diff
@@ -18,8 +22,13 @@ const PwdDate = ({ passwordCreated }: { passwordCreated: number }) => {
     day: 'numeric',
   }).format(new Date(passwordCreated));
 
+  const pwdDateFluent = getLocalizedDate(
+    passwordCreated,
+    LocalizedDateOptions.NumericDate
+  );
+
   return (
-    <FtlMsg id="security-password-created-date" vars={{ date: pwdDateText }}>
+    <FtlMsg id="security-password-created-date" vars={{ date: pwdDateFluent }}>
```

Comment:
This might be a question for @bcolsson, but wouldn't Fluent handle localization of the date if we passed in a `date` variable instead of a `string`?

According to Fluent's documentation :
> If the variable passed from the developer is a date and is used in a placeable, FTL will implicitly call a DATETIME function on it.

Reply:
I believe that's what the `getLocalizedDate` function is doing, providing a datetime that can be parsed by fluent:
https://github.com/mozilla/fxa/blob/3c3480e15b44ee7fd739281f42ca818c38d3840b/packages/fxa-payments-server/src/lib/formats.ts#L82-L96

Reply:
Right, I see that we're using this to pass localization options with the date. 

Equivalent to passing the options in more explicitly, as documented? @bcolsson would it be preferable to expose these options to localizers so they can have visibility on the date formatting? If not like below, then at least in a comment?
`today-is = Today is { DATETIME($date, month: "long", year: "numeric", day: "numeric") }`

@StaberindeZA this is non-blocking BTW - I'm somewhat hijacking your PR to get more details on how to handle date localization in the future :)

Reply:
In the case of things like correct date and number formatting, typically it is expected that the internationalization layer should handle the correct formatting for each locale without intervention by the linguists.

Reply:
That being said, I see your point regarding how a date is displayed. We could definitely add a comment if you wanted to include how it appears.